cpp/tests/io/parquet_test.cpp

```diff
@@ -4310,19 +4315,19 @@ TEST_F(ParquetReaderTest, BinaryAsStrings)
   cols.push_back(std::make_unique<cudf::column>(static_cast<cudf::column_view>(float_col)));
   cols.push_back(std::make_unique<cudf::column>(static_cast<cudf::column_view>(string_col)));
   cols.push_back(std::make_unique<cudf::column>(static_cast<cudf::column_view>(list_int_col)));
-  auto ouput = std::make_unique<table>(std::move(cols));
-  EXPECT_EQ(5, ouput->num_columns());
-  cudf_io::table_input_metadata ouput_metadata(*ouput);
-  ouput_metadata.column_metadata[0].set_name("col_other");
-  ouput_metadata.column_metadata[1].set_name("col_string");
-  ouput_metadata.column_metadata[2].set_name("col_float");
-  ouput_metadata.column_metadata[3].set_name("col_string2").set_output_as_binary(true);
-  ouput_metadata.column_metadata[4].set_name("col_binary").set_output_as_binary(true);
+  auto output = std::make_unique<table>(std::move(cols));
```

Comment:
Nitpick (and peripheral): I might have mentioned this before: Constructing the `output` table needn't be verbose:
```
auto output_view = table_view{{int_col, string_col, float_col, string_col, list_int_col}};
```

Reply:
This cleaned up a good deal of tests, thank you!